v4/source/channels.yaml

```diff
@@ -189,6 +189,42 @@
             // GetPublicChannelsByIdsForTeam
             channels, resp := Client.GetPublicChannelsByIdsForTeam(<TEAMID>, channelIds)
 
+  /channels/{channel_id}/timezones:
+    get:
+      tags:
+        - channels
+      summary: Get users timezones for a channel
+      description: |
+        Get users timezones for a channel.
```

Comment:
And this to `Get a list of timezones for the users who are in this channel.`

Reply:
done